zebra-chain/src/block/commitment.rs

```diff
@@ -123,9 +123,9 @@ impl Commitment {
                 }
             }
             // NetworkUpgrade::current() returns the latest network upgrade that's activated at the provided height, so
-            // on Regtest for heights above height 0, it returns NU5, and it's possible for the current network upgrade
-            // to be NU5 (or Canopy, or any network upgrade above Heartwood) at the Heartwood activation height.
-            (Canopy | Nu5, activation_height)
+            // on Regtest for heights above height 0, it returns NU6, and it's possible for the current network upgrade
```

Comment:
I think this has become outdated since it was written

```suggestion
            // on Regtest for heights above height 0, it could return NU6, and it's possible for the current network upgrade
```